kotsadm/pkg/snapshot/types/types.go

```diff
@@ -68,24 +68,31 @@ type Backup struct {
 }
 
 type BackupDetail struct {
-	Name             string           `json:"name"`
-	Status           string           `json:"status"`
-	VolumeSizeHuman  string           `json:"volumeSizeHuman"`
-	Namespaces       []string         `json:"namespaces"`
-	Hooks            []SnapshotHook   `json:"hooks"`
-	Volumes          []SnapshotVolume `json:"volumes"`
-	ValidationErrors []string         `json:"validationErrors,omitempty"`
-	Errors           []SnapshotError  `json:"errors"`
-	Warnings         []SnapshotError  `json:"warnings"`
+	Name            string           `json:"name"`
+	Status          string           `json:"status"`
+	VolumeSizeHuman string           `json:"volumeSizeHuman"`
+	Namespaces      []string         `json:"namespaces"`
+	Hooks           []SnapshotHook   `json:"hooks"`
+	Volumes         []SnapshotVolume `json:"volumes"`
+	Errors          []SnapshotError  `json:"errors"`
+	Warnings        []SnapshotError  `json:"warnings"`
+}
+
+type RestoreDetail struct {
+	Name     string          `json:"name"`
+	Phase    string          `json:"phase"`
+	Volumes  []RestoreVolume `json:"volumes"`
+	Errors   []SnapshotError `json:"errors"`
+	Warnings []SnapshotError `json:"warnings"`
 }
 
 type SnapshotHook struct {
 	Name          string          `json:"name"`
 	Namespace     string          `json:"namespace"`
-	Phase         string          `json:"pase"`
+	Phase         string          `json:"phase"`
```

Comment:
Does this change not break anything?

Reply:
i dont see anything using `. pase` but i dont know